tests/test_management.py

```diff
@@ -92,12 +93,15 @@ def test_transmit_courseware_task_success(fake_client, fake_catalog_client, capl
         get_catalog_courses=get_catalog_courses,
     )
 
+    track_selection_reverse_mock.return_value = '/course_modes/choose/course-v1:edX+DemoX+Demo_Course/'
+
     caplog.set_level(logging.INFO)
 
     UserFactory(username='C-3PO')
     enterprise_customer = EnterpriseCustomerFactory(
         catalog=1,
         name='Veridian Dynamics',
+        site__domain='supersuccessful.com'
```

Comment:
Prolly should stick with `example.com` -- this actually appears to be a registered domain name although nothing renders.